sql/core/src/test/resources/sql-tests/results/group-analytics.sql.out

```diff
@@ -275,6 +245,24 @@ NULL	2012	35000
 NULL	2013	78000
 
 
+-- !query
+SELECT course, year, SUM(earnings) FROM courseSales GROUP BY CUBE(course, year), ROLLUP(course, year) ORDER BY course, year
+-- !query schema
+struct<>
+-- !query output
+java.lang.UnsupportedOperationException
```

Comment:
This is a bad error message. Can we fail it earlier in the parser and give a clear error message?

Reply:
> parser

Done.  
After this merged, I will restart  https://issues.apache.org/jira/browse/SPARK-33229 Since TeraData. PostgresSQL, SQLServer, Oracle both support this way